xUnitTests/LoopringGraphTests/TestNFT.cs

```diff
@@ -67,5 +69,16 @@ public async void GetCollectionNFTs(string tokenAddress)
             var nfts = await service.GetCollectionNFTs(tokenAddress);
             Assert.NotEmpty(nfts);
         }
+
+        [Fact]
+        public async void GetNftHolders_WhenPassingValidNftId_ExpectListOfHolders()
+        {
+            string nftId = "0xf5ca5e167819d449270dae6fa22208085a2c03d9-0-0xcd56d4e1dba6edf425aa3e0f901d9f1645887539-0x33f5b90cdf6271026f4a0e4817de618268e6e391320a4ad834ea54fd87032fcb-2";
+            List<NftHolder> holders = await service.GetNftHolders(nftId);
+            Assert.NotEmpty(holders);
+
+            Assert.IsType<NftHolder>(holders[0]);
```

Comment:
Is it even possible for this to have another type? I mean when would this assertion fail? AFAICS never.

Reply:
It may be not necessary atm, but in the case that it's ever a change to this method or return type we have a test that will fail to remind us.

Reply:
Fixed tests